src/server/handlers/catch-all-handler.tsx

```diff
@@ -25,11 +25,15 @@ import { setForwardedHeaders } from "../utils/set-forwarded-headers";
 export default async (req: Request, res: Response) => {
   try {
     const activeRoute = routes.find(route => matchPath(req.path, route));
-    let auth = req.cookies ? cookie.parse(req.cookies).jwt : undefined;
+
+    let auth = req.headers.cookie
```

Comment:
Actual change here.

Reply:
That's so strange! TypeScript told me it was there!